meetings/2018-03-22.md

```diff
@@ -0,0 +1,151 @@
+* **Minutes Google Doc**: https://docs.google.com/document/d/1FGaeSlkiiM8sFGn0l_qLF0qEmXtlv-vYTQ8k4GbyjyM/edit
+
+## Present
+
+- Tierney Cyren (@bnb)
+- @hackygolucky (Tracy Hinds member)
+- @JemBijoux (Jem Bijoux member)
+- @chowdhurian (Manil Chowdhury member)
+- @obensource (Ben Michel member)
+- @tiriel (Benjamin Zaslavsky observer)
+- @williamkapke (William Kapke - Individual Membership Director)
+- @ZibbyKeaton (Zibby Keaton - observer)
+- @Bamieh (Ahmad Bamieh - observer)
+- @elainechan (Elaine Chan - observer)
+- @amiller-gh (Adam Miller – Member)
+- @dshaw (Dan Shaw - Member)
+- @joesepi (Joe Sepi - Member)
+- @codeekage (Agiri Abraham - observer)
+- @oe (Olivia - Member)
+
+## Agenda
+
+- nodejs/summit
+- Collaborator Summit Berlin 2018 - May 31st, June 1st 2018 #60
+- nodejs/admin
+- Travel Fund approval and being careful about funds available #99
+- nodejs/community-committee
+- Automate Meeting Participant History Document #267
+- Define Working Group expectations and create boilerplate #201
+- nodejs/mentorship
+- communication channel for the matched mentor/mentee 's first meeting #11
+
+## Initiative Review / Updates:
+
+### i18n
+- Not as much activity recently
+- Looking for others to share the leadership 
+- Folks in localization groups stepping up
+
+### Badges
+- not been able to develop actual content
+- there’s increased engagement on the repo
+- first meeting to be done soon
+
+### User Feedback
+- Great interest in public session
+- March 30th (9am PDT): getting together the tooling space folks to share (with some TSC and CommComm). General meet and greet.
+- Positive engagement so far!
+- Public User Feedback Session focused on Node.js Tooling: https://github.com/nodejs/user-feedback/issues/38
+
+### Mentorship
+- Tomorrow team meeting (9am PDT) if you’re interested in getting involved
+- Ahmad taking this on, Dan helping
+- Actively setting up the team
+- Need more individuals and more representation
+- Join the Mentorship team: https://github.com/nodejs/mentorship/issues/12
+- Dan needs access to calendar (add self to readme in admin repo as submitted pr)
+- (Mark tried to add, on todo list, didn’t quite get it done yet)
+
+## Agenda Topics
+
+### nodejs/summit: Collaborator Summit Berlin 2018 - May 31st, June 1st 2018
+
+- [Issue #60](https://github.com/nodejs/summit/issues/60)
+- Collaborator Summit Berlin 2018 - May 31st, June 1st 2018
+- Olivia added this point to the agenda (not in attendance initially)
+- Previously talked about preparing our session, will table temporarily and address on issue 
+- Start preparing if you are going to come; pairing with JSconf EU. Lots of collaborators there.
+
+### nodejs/admin: Travel Fund approval and being careful about funds available 
+
+- [Issue #99](https://github.com/nodejs/admin/issues/99)
+- Anybody on the call who is a CommComm member has privilege and responsibility to +1 travel fund requests
```

Comment:
wouldnt call it "responsibility to +1" because that implies they cant be vetoed... something like "privilege and responsibility to ratify"